lib/wallaby/dsl/finders.ex

```diff
@@ -13,6 +27,15 @@ defmodule Wallaby.DSL.Finders do
     do_find_elements(session, to_params(query))
   end
 
+  def any?(session, query) do
+    retry fn ->
+      case do_find_elements(session, to_params(query)) do
+        elements when length(elements) > 0 -> elements
```

Comment:
My initial thought was to just have `find/2` and `all/2` as finders for nodes. If you want to specify a count then you could add that as well.

`any?/` would be specifically for assertions. 

That seems like its not what you're going for though no?